### PR TITLE
Doc fixes

### LINKED ISSUE #436: [REQ] add ubuntu targets to docs
### What kind of request is this?

Other

### What is your request or suggestion?

document ubuntu targets such as `jammy`

### Are you willing to submit PRs to contribute to this feature request?

- [ ] Yes, I am willing to implement it.

### PR DESCRIPTION
**Pull request by cpuguy83**
See individual commits for details:

- Fix apt repo example
- Expand targets description
- Fix bad copy/paste for artifact permissions

Close #436

### CODE REVIEW

File: website/docs/targets.md [hunk 1]
```diff
@@ -2,12 +2,32 @@
 title: Targets
 ---
 
-This section provides an overview of the targets that Dalec supports. At this time, supported targets are `mariner2`, `azlinux3`, and `windowscross`.
-
-Many components, such as package dependencies and base images, are specific to a distro or a subset of distros. The dalec spec allows you to move these distro specific things into a `target`.
+DALEC is designed to support building artifacts for a number of different
+systems.
+DALEC refers to these in the the [spec](spec.md) as "targets".
```
Comment (DannyBrito) [inline website/docs/targets.md:7]:
```suggestion
DALEC refers to these in the [spec](spec.md) as "targets".
```

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Our goal is to provide a secure and reproducible way to build packages and conta
 
 - 🐳 No additional tools are needed except for [Docker](https://docs.docker.com/engine/install/)!
 - 🚀 Easy to use declarative configuration that provides reproducible builds
-- 📦 Build packages and/or containers for [Azure Linux](https://github.com/microsoft/azurelinux) 2 and 3 (formerly known as CBL-Mariner)
+- 📦 Build packages and/or containers for a number of different [targets](https://github.com/azure.github.io/dalec/targets)
```
Comment (sozercan) [inline README.md:14]: Users might not know what a target is, would we want to the operating systems? 

```suggestion
- 📦 Build packages and/or containers for a number of different [targets](https://github.com/azure.github.io/dalec/targets) such as Ubuntu and Azure Linux
```

File: website/docs/targets.md (hunk 1 above)
Comment (cpuguy83) [reply website/docs/targets.md]: Fixed.

File: README.md (hunk 2 above)
Comment (cpuguy83) [reply README.md]: I like it, thanks!
Updated.

File: website/docs/targets.md [hunk 3]
```diff
@@ -2,12 +2,32 @@
...
-Many components, such as package dependencies and base images, are specific to a distro or a subset of distros. The dalec spec allows you to move these distro specific things into a `target`.
+DALEC is designed to support building artifacts for a number of different
+systems.
+DALEC refers to these in the [spec](spec.md) as "targets".
+When executing a build with Docker these targets can be specified with the
+`--target=<target>` flag.
 
 ## Available Targets
 
+DALEC includes a number of built-in targets that you can either use in your spec.
```
Comment (MorrisLaw) [inline website/docs/targets.md:13]: oh wow, I was just referencing this too. Thanks for adding more targets to the list!

**Review by MorrisLaw — APPROVED** [review]
:shipit:

**Review by DannyBrito — APPROVED** [review]
lgtm

**Pull request merged**